llm_studio/python_configs/text_causal_classification_modeling_config.py

```diff
@@ -54,11 +56,19 @@ def __post_init__(self):
 
         self._possible_values["num_classes"] = (1, 100, 1)
 
+        self._visibility["system_column"] = -1
+        self._visibility["parent_id_column"] = -1
+        self._visibility["text_system_start"] = -1
+        self._visibility["text_prompt_start"] = -1
+        self._visibility["text_answer_separator"] = -1
+        self._visibility["add_prompt_answer_tokens"] = -1
```

Comment:
why remove all these settings? might still be useful

Reply:
Yeah, it is a tradeoff between simplicity and customizability. 
I'd argue that system prompt can at least be removed as it can fully be replaced by Text prompt start in absense of conversation chaining. So, we might want to reduce to pre prompt and post prompt.

Reply:
I would still keep start and end token options for example

Reply:
So, keep `text_prompt_start` and `add_eos_token_to_prompt` ? I would agree with that. 

Adding an EOS after answer makes no sense, and I would still argue in favor of removing the system column. One could still select multiple prompt columns.

Reply:
I would rather keep `text_answer_separator`, but also `text_prompt_start  `and `add_eos_token_to_prompt `
removing sys is fine to me

Reply:
Ok, sounds good. Will change it accordingly.